Framework/HistogramData/test/HistogramTest.h

```diff
@@ -932,4 +934,37 @@ class HistogramTest : public CxxTest::TestSuite {
   }
 };
 
+class HistogramTestPerformance : public CxxTest::TestSuite {
+public:
+  static HistogramTestPerformance *createSuite() {
+    return new HistogramTestPerformance;
+  }
+  static void destroySuite(HistogramTestPerformance *suite) { delete suite; }
+
+  HistogramTestPerformance() : copy(histSize, LinearGenerator(0, 2)) {
+    for (size_t i = 0; i < nHists; i++)
+      hists.push_back(Histogram(BinEdges(histSize, LinearGenerator(0, 1))));
+
+    histsCopy = hists;
```

Comment:
Maybe add some comment that explains this. If I remember correctly it was to avoid benchmarking de-allocation?